eZ/Publish/Core/Search/Legacy/Content/Mapper/FullTextMapper.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Search\Legacy\Content\Mapper;
+
+use eZ\Publish\Core\Search\Common\FieldRegistry;
+use eZ\Publish\Core\Search\Legacy\Content\FullTextData;
+use eZ\Publish\SPI\Persistence\Content;
+use eZ\Publish\SPI\Persistence\Content\Type;
+use eZ\Publish\SPI\Search\FieldType;
+use eZ\Publish\SPI\Persistence\Content\Type\Handler as ContentTypeHandler;
+use eZ\Publish\Core\Search\Legacy\Content\FullTextValue;
+
+/**
+ * FullTextMapper maps Content object fields to FullTextValue objects which are searchable and therefore can be indexed by the legacy search engine.
```

Comment:
Long lines should be wrapped, I think somewhere in our CS we have that it should be done at 100th column :)